Three/include/CGAL/Three/Viewer_interface.h

```diff
@@ -43,7 +44,12 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
   Q_OBJECT
 
 public:
-  enum OpenGL_program_IDs { PROGRAM_WITH_LIGHT,
+ QPainter *painter;
+ TextRenderer *textRenderer;
+ //!Returns true if the primitive ids are displayed
```

Comment:
`Viewer_interface` is an *interface*. It should be an empty class, with pure virtual methods, and nothing else!